test/compile/selection/scales.test.ts

```diff
@@ -36,11 +37,11 @@ describe('Selection + Scales', function() {
             },
             color: {
               field: "symbol", type: "nominal",
-              scale: {domain: {selection: "brush2"}}
+              scale: {domain: {selection: "brush2"} as Domain}
```

Comment:
How bizarre -- I don't understand why my VSCode didn't show this as an error, nor why it wasn't caught by Travis. The cast looks fine to me -- I had wanted to test a warning in case the spec was invalid. `brush3` can include an encoding or field, that's testing another warning.